cumulusci/tasks/bulkdata/tests/test_extract.py

```diff
@@ -339,35 +335,13 @@ def test_import_results__autopk(self, csv_mock):
                 ),
                 mock.call(
                     connection=task.session.connection.return_value,
-                    table="Opportunity_sf_ids",
+                    table=mapping.get_sf_id_table(),
                     columns=["sf_id"],
                     record_iterable=csv_mock.return_value,
                 ),
             ]
         )
 
-    def test_import_results__no_columns(self):
```

Comment:
Removed this test because I changed the behavior: it is perfectly legal, and occasionally desirable, to extract an object with no columns (save the automatically-included `Id`).